### ISSUE TITLE
There is probably a case for the lapack bindings to exist independently of this package

### ISSUE THREAD

**Issue by dataPulverizer** [DlangScience/scid#50]
There is probably a case for the lapack bindings to exist independently of this package

**Comment by John-Colvin** [comment]
Could be done as a dub sub-package?

**Comment by dataPulverizer** [comment]
I didn't know about sub-dub, just looked at it in the manual. You guys know more about the DUB packaging system in which I am still a novice but my thinking is that lapack can exist as a library with cblas binding as a dependency that DUB can manage.

**Comment by kyllingstad** [comment]
I agree, it would probably be better if it were a separate package under DlangScience, like the BLAS bindings. I'm not too thrilled about the sub-package idea. [Deimos](https://github.com/D-Programming-Deimos) is also an option.

**Comment by John-Colvin** [comment]
I agree, it should be a separate package entirely
